README.md

```diff
@@ -37,14 +37,15 @@ cp .config.json config.json
 ```
 
 Change parameter in `config.json` e.g.
-```
+```json
 {
-  "host": "http://mattermost.example.com:8065", # The URL of your Mattermost server
-  "port": 8505, # The port matterpoll should listen to requests
-  "token": "9jrxak1ykxrmnaed9cps9i4cim", # The Token created my Mattermost
+  "host": "http://mattermost.example.com:8065",  // The URL of your Mattermost server
+  "listen": ":8505",  // The address:port to listen on
+  "address": "",  // Optional address to bind to and isten on
+  "token": "9jrxak1ykxrmnaed9cps9i4cim",  // The Token created my Mattermost
   "user": {
-   "id": "bot",          # The username of an existing Mattermost account
-   "password": "botbot"  # The password of an existing Mattermost account
+   "id": "bot",  // The username of an existing Mattermost account
+   "password": "botbot"  // The password of an existing Mattermost account
```

Comment:
The // commands make it look quite bad on github. Do you know if there is any way to remove the red background?